### DISCUSSION TITLE
How to remove tab "Products & services"

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#14559] [How to] [answered]
How to remove tab "Products & services" from Account module?

**Comment by vovpff** [comment, marked as answer]
I'm not sure but pls check config of this module at path `/config/modules/`

**Reply by rskrzypczak** [reply]
@vovpff is right.
more specifically: `config\Modules\Accounts.php` 
```
/** Show summary products services bookmark */
public static $SHOW_SUMMARY_PRODUCTS_SERVICES = true;
```

**Reply by TonyM15** [reply]
Thank you gents. Getting more and more impressed by default functionalities.

**Reply by TonyM15** [reply]
By the way where to register respective class so I can get similar config for custom module?

**Reply by Giorgos-Hellene** [reply]
> 
> 
> @vovpff is right.
> more specifically: `config\Modules\Accounts.php`
> 
> ```
> /** Show summary products services bookmark */
> public static $SHOW_SUMMARY_PRODUCTS_SERVICES = true;
> ```

Could you tell us the same about Opportunities ?

**Reply by vovpff** [reply]
What you want for Opportunities? You want to hide this tab?

**Reply by Giorgos-Hellene** [reply]
@vovpff Yes exactly ! if it is possible

**Reply by vovpff** [reply]
Hm. If you need to hode Opportunity tab in Accounts you need to go to telation manager, select Accounts and click hide button on Opportunity block.

**Reply by Giorgos-Hellene** [reply]
@vovpff i am interesting the same to opportunities as accounts. So i want to hide Products & services on opportunites module.
![εικόνα](https://user-images.githubusercontent.com/50912403/115995870-a2a8d700-a5e5-11eb-89a5-e99560bdfeac.png)

**Reply by vovpff** [reply]
Ok, lets check config files?

**Reply by Giorgos-Hellene** [reply]
Could you tell me which is the right file to hide Products & services on opportunites module ?

**Reply by vovpff** [reply]
Is it difficult to check it by yourself?

`/config/modules/SSalesProcesses.php`

String

```
SHOW_SUMMARY_PRODUCTS_SERVICES
```

Should be false to hide

**Reply by Giorgos-Hellene** [reply]
@vovpff yes it was difficult for me because i didn't know which was the right file. I was searching something like Opportunities and there isn't. 
Yes, this is that i need.
Thank you for your help and your time !

**Reply by vovpff** [reply]
No problem
